packages/data-table/src/styles/DataTable.css.ts

```diff
@@ -0,0 +1,56 @@
+import { vars } from '@launchpad-ui/vars';
+import { style } from '@vanilla-extract/css';
+import { recipe } from '@vanilla-extract/recipes';
+
+const focusVisible = style({
+  outline: 'none',
+  ':focus-visible': {
+    boxShadow: `0 0 0 2px ${vars.color.shadow.interactive.primary}, 0 0 0 4px ${vars.color.shadow.interactive.focus}`,
+  },
+});
+
+const border = style({
+  border: `1px solid ${vars.color.border.ui.primary}`,
+});
+
+const active = style({
+  background: vars.color.bg.interactive.secondary.active,
+});
+
+const table = style([
+  border,
+  {
+    borderCollapse: 'collapse',
+    background: vars.color.bg.ui.primary,
+  },
+]);
+
+const cell = style([
+  focusVisible,
+  {
+    padding: `${vars.spacing[500]} ${vars.spacing[600]}`,
+    cursor: 'default',
+  },
+]);
```

Comment:
Loving the simplicity of style composition with VE. Could see us having a package to supply base styles such as focus-visible to minimize duplication and have a single source of truth.